src/CodeStyle/Core/Analyzers/DiagnosticHelper.cs

```diff
@@ -0,0 +1,248 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CodeStyle
+{
+    internal static class DiagnosticHelper
```

Comment:
>DiagnosticHelper [](start = 26, length = 16)

Same suggestion here - use linked files instead of cloning.

Reply:
It should be removable now actually; will check.